Makefile

```diff
@@ -29,7 +29,7 @@ define run-gradle-tasks
 endef
 
 .PHONY: check
-check: license-verification
```

Comment:
@RingerJK was this change intentional? It disabled the license check on CI.

Reply:
seems that was done by mistake, let me address it to PR

Reply:
https://github.com/mapbox/mapbox-navigation-android/pull/4252